### DISCUSSION TITLE
v4 roadmap

### DISCUSSION THREAD

**Discussion by Hideman85** [BitPhinix/slate-yjs#307] [Ideas]
Hi @BitPhinix

I'm wondering if there is a roadmap for the v4?
I'm also willing to work on the Y.Text format for marks & inline node and wondering if we could have a branch for?

I'm looking forward to the next version

**Comment by BitPhinix** [comment]
You can see the scope of the v4 milestone here: https://github.com/BitPhinix/slate-yjs/milestone/1

There isn't an exact timeframe for a stable release yet, but an active development branch here: https://github.com/BitPhinix/slate-yjs/tree/next. Please keep in mind that the code in there will likely change a lot over the next few weeks

**Reply by Hideman85** [reply]
Your start looks nice I'm gonna see to invest work on my sparse time ;)
I do think the most important one at first is the linearisation of text & inline elements with the use of formats.

**Reply by BitPhinix** [reply]
Yeah, i see it exactly your way. The current implementation on there isn't finished and definitely has some bugs but I think it solves this issue nicely
